cpp/src/arrow/array-test.cc

```diff
@@ -1154,40 +1154,44 @@ TEST_F(TestBinaryBuilder, TestScalarAppend) {
     }
   }
 }
- 
+
 TEST_F(TestBinaryBuilder, TestCapacityReserve) {
-  vector<string> strings = {"aaaaa", "bbbbbbbbbb", "ccccccccccccccc", "dddddddddddddddddddd", "eeeeeeeeee"};
+  vector<string> strings = {"aaaaa", "bbbbbbbbbb", "ccccccccccccccc", "dddddddddd"};
```

Comment:
In the future, you can run `make format` (which uses clang-format) to fix these long lines without having to make code changes